src/odemis/gui/util/test/img_test.py

```diff
@@ -1083,12 +1090,17 @@ def setUp(self):
         sem_stream_pj.mpp.value = 1e-6
 
         self.streams = [fluo_stream_pj, sem_stream_pj]
-        self.min_res = (623, 432)
+        self.image_ration = [[fluo_stream_resolution[0] / fluo_stream_resolution[1]],
```

Comment:
Typo: ration -> ratio.

Maybe just run one more time all the test cases, to check everything is in order 😉